salt/orchestrate/edx/backup.sls

```diff
@@ -102,6 +124,35 @@ execute_enabled_backup_scripts:
         - salt: deploy_backup_instance_to_{{ ENVIRONMENT }}
         - salt: mount_backup_drive
 
+unmount_backup_drive:
+  salt.function:
+    - tgt: 'G@roles:backups and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - mount.unmounted
+    - kwarg:
+        name: /backups
+        device: /dev/xvdb
+    - require:
+        - salt: execute_enabled_backup_scripts
+
+detach_backup_volume:
+  salt.function:
+    - tgt: 'G@roles:backups and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: saltutil.runner
+    - arg:
+        - cloud.action
+    - kwarg:
+        func: boto_ec2.detach_volume
+        kwargs:
+          tags: {{ backup_volume_name }}
+          instance_id: {{ instance_id }}
```

Comment:
instance_id is set based on value returned from `boto_ec2.find_instances('name=backup-..)`
[Line#69](https://github.com/mitodl/salt-ops/blob/e1d6d884fcb7476d870814c4c20999a863459692/salt/orchestrate/edx/backup.sls#L69)